codecov.yml

```diff
@@ -0,0 +1,22 @@
+codecov:
```

Comment:
Is there any way for this file to be named `.codecov.yml` or be hidden in one of our existing subdirectories somehow? All the other similar configurations are dotfiles, and I'd prefer `ls -l` in the top directory to show just MFEM-relevant sources.

Reply:
I couldn't find this in the doc... but then I looked in spack repo... and they have it hidden. Let's try!

Reply:
Apparently it didn't work. Spack is using the codecov action, maybe that's why. I’ll dig into it.